pkg/tsdb/cloudmonitoring/annotation_query.go

```diff
@@ -19,21 +19,46 @@ func (e *CloudMonitoringExecutor) executeAnnotationQuery(ctx context.Context, ts
 		return nil, err
 	}
 
-	queryRes, resp, _, err := queries[0].run(ctx, tsdbQuery, e)
+	queryRes, resp, executedQuery, err := queries[0].run(ctx, tsdbQuery, e)
 	if err != nil {
 		return nil, err
 	}
 
+	err = queries[0].parseResponse(queryRes, resp, executedQuery)
+	if err != nil {
+		queryRes.Error = err
+	}
+
 	metricQuery := firstQuery.Model.Get("metricQuery")
 	title := metricQuery.Get("title").MustString()
 	text := metricQuery.Get("text").MustString()
 	tags := metricQuery.Get("tags").MustString()
+
+	transformToDataframes(queryRes, title, tags, text)
```

Comment:
transforming the table to dataframes should be done after parseToAnnotations is called

Reply:
This is now deleted, all the work is happening inside `parseToAnnotations` method.